src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/Converters/Converters.LambdaMultiConverter.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+#nullable disable
+
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Utilities
+{
+    // Copied from CPS Converters.cs
+    internal static partial class Converters
```

Comment:
Same comment as above.

Reply:
This is only going to be used in the property pages and relates to Converters. I'm of the opinion that it should stay where it is. This is the same thing that CPS does